src/FlyCube/QueryHeap/DXRayTracingQueryHeap.cpp

```diff
@@ -14,7 +14,7 @@ DXRayTracingQueryHeap::DXRayTracingQueryHeap(DXDevice& device, QueryHeapType typ
     auto desc = CD3DX12_RESOURCE_DESC::Buffer(count * sizeof(uint64_t));
     desc.Flags = D3D12_RESOURCE_FLAG_ALLOW_UNORDERED_ACCESS;
     m_device.GetDevice()->CreateCommittedResource(&CD3DX12_HEAP_PROPERTIES(D3D12_HEAP_TYPE_DEFAULT),
-                                                  D3D12_HEAP_FLAG_NONE, &desc, D3D12_RESOURCE_STATE_UNORDERED_ACCESS,
+                                                  D3D12_HEAP_FLAG_NONE, &desc, D3D12_RESOURCE_STATE_COMMON,
```

Comment:
Could you also change `D3D12_RESOURCE_STATE_UNORDERED_ACCESS` to `D3D12_RESOURCE_STATE_COMMON` in `DXCommandList::ResolveQueryData`?

Reply:
sure